### PR TITLE
AGENT-287: For agent mirror testing force disconnect from masters

### PR DESCRIPTION
**Pull request by bfournie**
This simulates a disconnect from the masters when using the agent with mirroring. It sets a bogus /etc/hosts entry so that the local mirror must be used.  This is currently needed since we must have a valid connection to create the mirror and create the iso so we can't break the connection until after that's done.

### CODE REVIEW

**Review by pawanpinjarkar — COMMENTED** [review]
/lgtm

Can you also mention the Jira number in the commit message and PR title?

File: agent/05_agent_create_cluster.sh [hunk 1]
```diff
@@ -39,6 +39,31 @@ function attach_agent_iso() {
...
+     ssh_opts=(-o 'StrictHostKeyChecking=no' -q core@${node_ip})
+
+     until ssh "${ssh_opts[@]}" "[[ -f /etc/hosts ]]"
+     do
+       echo "Waiting for $node_name to set remote host disconnect "
+       sleep 30s;
+     done
+
+     # Set a bogus entry in /etc/hosts to break remote access
+     ssh "${ssh_opts[@]}" "echo '125.12.15.15 quay.io' | sudo tee -a /etc/hosts "
```
Comment (rwsu) [inline agent/05_agent_create_cluster.sh:62]: The mirroring command I see from dev-scripts is:
`oc adm release mirror --insecure=true -a /opt/dev-scripts/pull_secret.json --from registry.ci.openshift.org/ocp/release:4.11.0-0.nightly-2022-07-13-131410 --to-release-image virthost.ostest.test.metalkube.org:5000/localimages/local-release-image:4.11.0-0.nightly-2022-07-13-131410 --to virthost.ostest.test.metalkube.org:5000/localimages/local-release-image 
`
Shouldn't there be an entry for registry.ci.openshift.org?

Comment (bfournie) [reply agent/05_agent_create_cluster.sh:62]: Yes there should be an entry for registry also, added it.

**Review by rwsu — COMMENTED** [review]
/lgtm

**Pull request merged**